main.py

```diff
@@ -1,7 +1,37 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    for genre_str in ("Western", "Action", "Dramma"):
+        Genre.objects.create(name=genre_str)
+
+    for actor_str in (
+            "George Klooney",
+            "Kianu Reaves",
+            "Scarlett Keegan",
+            "Will Smith",
+            "Jaden Smith",
+            "Scarlett Johansson"
+    ):
+        first_name, last_name = actor_str.split()[0], actor_str.split()[1]
```

Comment:
```suggestion
        first_name, last_name = actor_str.split()
```
No need to split two times